### ISSUE TITLE
Can't control chrome if --enable-automation is not set

### ISSUE THREAD

**Issue by pu369** [go-rod/rod#232]
rod version v0.64.0

```go
package main

import (
	"github.com/go-rod/rod"
	"github.com/go-rod/rod/lib/launcher"
)

func main() {
	url := launcher.NewUserMode().MustLaunch()
	_ = rod.New().ControlURL(url).MustConnect()
}
```

Error:

```
panic: {"code":-32000,"message":"Command line not returned because --enable-automation not set.","data":""}
```

**Comment by ysmood** [comment]
`launcher.NewUserMode().Delete("disable-blink-features").MustLaunch()`

**Comment by ysmood** [comment]
released the new version `v0.64.1`, should have fixed this bug. Please upgrade your rod version.
